### PR TITLE
Replace ip051U7Rvds -> KsOIlDT145A on hourofcode.com index page

### PR DESCRIPTION
**Pull request by joshlory**
@tanyaparker to confirm, the new video is the ["Calling All Teachers - WORLDWIDE"](https://www.youtube.com/embed/KsOIlDT145A) — is that correct?

### CODE REVIEW

**Comment by joshlory** [comment]
We'll need to cache-bust after this ships for the new changes to be reflected on hourofcode.com.

**Comment by joshlory** [comment]
Should we pull https://hourofcode.com/us/images/fit-1500/homepage_0.jpg out of the hero image rotation, now that the video is gone?

image

**Comment by joshlory** [comment]
@breville PTAL!

**Comment by tanyaparker** [comment]
Yup! We're removing all athletes focus.

**Comment by joshlory** [comment]
@breville I've also removed the athlete full width header image.  Do you mind taking another quick look when you get a chance?

File: pegasus/sites.v3/hourofcode.com/public/index.haml [hunk 1]
```diff
@@ -47,6 +47,16 @@ social:
     margin: 0px;
     padding: 0px;
   }
+  #fullwidth0, #fullwidth1, #fullwidth2, #fullwidth3, #fullwidth4 {
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/index.haml]: Could also use a CSS class here.

Comment (joshlory) [reply pegasus/sites.v3/hourofcode.com/public/index.haml]: Good call, fixed in 62747a5e377538ca9837e47cfab1203de3c03579.

**Pull request merged**